controllers/grid/files/signoff/form/FileAuditorForm.inc.php

```diff
@@ -7,7 +7,7 @@
  * Copyright (c) 2003-2015 John Willinsky
  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
  *
- * @class CopyeditingUserForm
+ * @class FileAuditorForm
  * @ingroup controllers_grid_files_signoff
  *
  * @brief Form to add files to the copyediting files grid
```

Comment:
I don't think we still need this form right? It's used by the signoff grid only...